docs/guides/databases/mariadb/backup-mariadb-mysql-to-object-storage-with-restic/index.md

```diff
@@ -1,16 +1,18 @@
 ---
```

Comment:
@andysh-uk I'm going to leave some comments that just explain my changes. They aren't change requests--I just wanted to show my reasoning. If you think that one or more of the changes goes against the spirit of the guide, please let me know!